src/Pokemon.java

```diff
@@ -0,0 +1,109 @@
+class Pokemon {
+
+    private String type;
+    protected static String name;
+    protected int hp;
+    protected String level;
+    private String specialAttack;
+    private String move;
+
+
+    public Pokemon (String type, String name, int hp, String level, String specialAttack, String move) {
+        this.type = type;
+        this.name = name;
+        this.hp = hp;
+        this.level = level;
+        this.specialAttack = specialAttack(specialAttack);
```

Comment:
Waarom sla je specialAttack op in een klassevariabele? Je gebruikt de klassevariabele nooit, het heeft ook geen getters of setters.
De `specialAttack()` methode is wel public, dus die kan gewoon gebruikt worden in Main.